recipes/netpbm/config.mk

```diff
@@ -0,0 +1,77 @@
+DEFAULT_TARGET = nonmerge
+BUILD_FIASCO = Y
+LD = $(CC)
+LINKERISCOMPILER=Y
+LINKER_CAN_DO_EXPLICIT_LIBRARY=Y
+INTTYPES_H = <inttypes.h>
+HAVE_INT64 = Y
+WANT_SSE = N
+CC_FOR_BUILD = $(CC)
+LD_FOR_BUILD = $(LD)
+CFLAGS_FOR_BUILD = $(CFLAGS_CONFIG)
+LDFLAGS_FOR_BUILD = $(LDFLAGS)
+WINDRES = windres
+INSTALL = $(SRCDIR)/buildtools/install.sh
+STRIPFLAG = -s
+SYMLINK = ln -s
+MANPAGE_FORMAT = nroff
+AR = ar
```

Comment:
Use `${AR}` from environment.

Reply:
This kind of replacement won't work. If it is changed to `${AR}`, makefile won't recognizes it because `${**}` will only be replaces manually by the `build.sh` script. Replace by conda's `$(AR)` will lead to recursive variable error. So, if conda sets `AR` variable (and `RANLIB`) the best thing we can do is simply delete these lines, forcing the make code look to the environment for these variables.

Reply:
> So, if conda sets AR variable (and RANLIB) the best thing we can do is simply delete these lines, forcing the make code look to the environment for these variables.

Works for me. You could also make it `AR ?= ar`

Reply:
Interesting. I specifically tried both replacements and both resulted in recursive errors. Using `?=` works for me, including for the `CC` flag I deleted to use the conda's `CC` var. I will replace these 3 variables (CC, AR and RANLIB) by this sintax.